include/emitc/emitc_tosa.h

```diff
@@ -49,6 +49,72 @@ inline Src add(Src x, Src y) {
   return emitc::add<Src>(x, y);
 }
 
+// MulOp
+template <typename Src>
+inline Src mul(Src x, Src y, int32_t shift) {
+  assert(shift == 0); // shift has to be 0 if datatype is not int32_t
+  return emitc::mul(x, y);
+}
+
+template <>
+inline Tensor0D<int32_t> mul(Tensor0D<int32_t> x, Tensor0D<int32_t> y,
```

Comment:
This won't work as expected, this function should be called for tensors with element type int32_t no matter what dimension.